src/views/PackageDetail.vue

```diff
@@ -50,7 +50,7 @@
             <div class="column is-4">
               <nav class="panel">
                 <p class="panel-heading">
-                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'parachute-box']" />Open this package
+                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'parachute-box']" />Use Module
```

Comment:
```suggestion
                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'parachute-box']" />Use this module
```

I do agree that switching to "module" universally is for the better. It's just a shame that the icon doesn't completely match with the word anymore.